src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -19,6 +19,19 @@ import { Component } from 'src/libs/wrapped-components'
 
 
 const styles = {
+  page: {
+    display: 'flex', flexDirection: 'column', height: '100%', flexGrow: 1
+  },
+  workspaceNameContainer: {
+    display: 'flex', flexDirection: 'column',
+    paddingLeft: '4rem', overflowX: 'hidden', marginRight: '0.5rem'
```

Comment:
nit: I think `min-width: 0` would have the same effect as this overflow, and might be clearer in intent.